llm/ggml.go

```diff
@@ -144,38 +142,22 @@ func DecodeGGML(r io.ReadSeeker) (*GGML, error) {
 		return nil, errors.New("invalid file magic")
 	}
 
-	model, err := c.Decode(&ro)
+	model, err := c.Decode(rs)
 	if errors.Is(err, io.EOF) {
 		// noop
 	} else if err != nil {
 		return nil, err
 	}
 
+	offset, err := rs.Seek(0, io.SeekCurrent)
```

Comment:
just confirming, the offset returned here is cumulative the same way as the custom implementation was?

Reply:
that's correct. there's no functional difference between this and the previous implementation in terms of results